pkgs/top-level/perl-packages.nix

```diff
@@ -7727,17 +7727,43 @@ let self = _self // overrides; _self = with self; {
       description = "Embed a Perl interpreter in the Apache HTTP server";
     };
   };
-
   Mojolicious = buildPerlPackage {
```

Comment:
You need to prefix the attribute set with `rec`, otherwise `name` appears undefined in the url